lib/vm_service_wrapper.dart

```diff
@@ -0,0 +1,268 @@
+import 'dart:async';
+import 'package:vm_service_lib/vm_service_lib.dart';
+
+class VmServiceWrapper implements VmService {
+  VmServiceWrapper(this._vmService);
+
+  VmServiceWrapper.fromNewVmService(
+      Stream<dynamic> /*String|List<int>*/ inStream,
+      void writeMessage(String message),
+      {Log log,
+      DisposeHandler disposeHandler}) {
+    _vmService = new VmService(inStream, writeMessage,
+        log: log, disposeHandler: disposeHandler);
+  }
+
+  VmService _vmService;
+  final Set<String> _activeStreams = Set();
+  final Set<Future<Object>> _activeFutures = Set();
+  Completer<bool> noPendingFutures = new Completer<bool>();
+
+  @override
+  Future<Breakpoint> addBreakpoint(String isolateId, String scriptId, int line,
+          {int column}) =>
+      _trackFuture(
+          _vmService.addBreakpoint(isolateId, scriptId, line, column: column));
+
+  @override
+  Future<Breakpoint> addBreakpointAtEntry(
+          String isolateId, String functionId) =>
+      _trackFuture(_vmService.addBreakpointAtEntry(isolateId, functionId));
+
+  @override
+  Future<Breakpoint> addBreakpointWithScriptUri(
+          String isolateId, String scriptUri, int line, {int column}) =>
```

Comment:
don't use => when the method body wraps. It makes the indentation a bit weird and the code hard to read.
use `{` and `return` instead.

Reply:
Done.